pkg/network/ovn_kubernetes.go

```diff
@@ -246,6 +259,7 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 func bootstrapOVNConfig(kubeClient client.Client) (*bootstrap.OVNConfigBoostrapResult, error) {
 	ovnConfigResult := &bootstrap.OVNConfigBoostrapResult{
 		GatewayMode:            OVN_SHARED_GW_MODE,
+		DPUMode:                OVN_DPU_MODE_FULL,
```

Comment:
This makes it sound like the default is "full DPU usage" rather than "full ovnkube-node functionality". Can you change "dpu mode" to "node mode" everywhere? (template variable names, golang variable names, struct fields, const names)

Reply:
Done